jest.config.js

```diff
@@ -2,6 +2,11 @@ module.exports = {
   moduleNameMapper: {
     "ui/(.*)": "<rootDir>/src/ui/$1",
   },
-  // If you want to run all of the old tests then remove this ignore pattern
-  modulePathIgnorePatterns: ["src/devtools"],
+  testEnvironment: "jsdom",
+  testMatch: [
+    "**/__tests__/**/*.[jt]s?(x)",
+    "**/?(*.)+(test).[jt]s?(x)",
```

Comment:
☝️  this is jest's [default test matcher](https://jestjs.io/docs/configuration#testmatch-arraystring), but without `.spec.js`

Reply:
Might be worth removing my comment below and adding a comment pointing to this PR to explain the reasoning.